src/bindings/pygaia/classification/classificationtask.py

```diff
@@ -172,10 +179,12 @@ def run(self, className, outfilename, param, dsname, gtname, evalconfig):
                                                                                               param['classifier'],
                                                                                               param['preprocessing']))
                 log.info('    PID: %d, parameters: %s' % (os.getpid(), json.dumps(param)))
+                
+                if jobidx and jobcount:
+                    log.info('    Job: %.*d/%d' % (jobdigits, jobidx, jobcount))
```

Comment:
In AcousticBrainz I'd like to be able to be able to set a callback function to get called each time an evaluation finishes. This would let us report status/progress of the training to users.
At the very least I'd like to have the iteration number and total number of jobs. A useful addition would be total time elapsed and time estimated to complete. I'm not sure if it's useful to also provide information about what combination of parameters was used in the just finished iteration (e.g. preprocessing, C, gamma values)

Reply:
I guess this wouldn't happen in classificationtask, but instead in the manager, since that would be in the same process space as the tool that runs our classification: https://github.com/metabrainz/acousticbrainz-server/blob/3d62f515f3321874f3ddd1b3625fe6e21c8ac3b7/dataset_eval/gaia_wrapper.py#L47